psh/ls/ls.c

```diff
@@ -16,6 +16,7 @@
 #include <grp.h>
 #include <pwd.h>
 #include <stdint.h>
+#include <stdbool.h>
```

Comment:
Using stdbool.h is compliant with C99. We used to write our code according to C89 convention. Currently, it is acceptable to use C99 specific features. However I would discourage you from using bool here, as we have not used in any other place in psh. But, as I said it is not stated clearly if we can use those, so it is a matter of taste. I personally don't like it.

Reply:
Thanks, I wasn't sure about it. I will try to write C89 compliant code then.